isis/src/control/objs/ControlNetVitals/ControlNetVitals.cpp

```diff
@@ -0,0 +1,245 @@
+#include "ControlNetVitals.h"
+
+#include <QList>
+
+#include "IException.h"
+#include "IString.h"
+#include "ControlNet.h"
+#include "ControlPoint.h"
+#include "ControlMeasure.h"
+
+namespace Isis {
+
+  ControlNetVitals::ControlNetVitals(ControlNet *cnet) {
+    m_controlNet = cnet;
+    validate();
+  }
+
+  ControlNetVitals::~ControlNetVitals() {
+  }
+
+  bool ControlNetVitals::hasIslands() {
+    // Replace this with graph call!!!$!@$!@$!@$#@%#@$%#@
+    return true;
+  }
+
+  int ControlNetVitals::numIslands() {
+    // replace this with graph call!#@$!#%#@%*($#)
+    return 1;
+  }
+
+  QList<QString> ControlNetVitals::getIslands() {
```

Comment:
This will need to return a QList of QLists of QStrings, the internal lists are the islands.